abi.html

```diff
@@ -5389,9 +5389,11 @@
 	  // Corresponding operator(): _ZNK1SIiE1xMUlvE_clEv
 </pre></code>
 
-
-
-
+<p>
+In a generic lambda, uses of <code>auto</code> in the parameter list
+are mangled as the underlying artificial template type parameter.  If
+the lambda is a template type argument, this can lead to manglings
+that superficially appear recursive.
```

Comment:
I would use "corresponding" instead of "underlying" here.

The second sentence begs for clarification.  Maybe instead:

> This is never ambiguous with a lambda parameter whose type is an enclosing template type parameter because lambdas are never mangled in a dependent context (because they are forbidden from appearing in function signatures).  A template-param in a lambda-sig can only ever refer to a template parameter of a generic lambda.